os-fpga/FOEDAG
# Clear button (Close #506)

### Motivate of the pull request
Close https://github.com/os-fpga/FOEDAG/issues/506

### Describe the technical details
Fixed compiler internal state. Clean action doesn't have green/red mark anymore.

### Testaces

Steps:

    1. Run some project compilation.
    2. Wait until it finished successfully.
    3. Press Clean for Placement.

Expected results:

    * Clean was performed.
    * No green mark near the clean action.
    * No any green marks for Placement and below.

![image](https://user-images.githubusercontent.com/95262932/178760489-8bcbb73d-dfd4-49ae-896b-575add0af0ac.png)
